presto-docs/src/main/sphinx/release/release-327.rst

```diff
@@ -0,0 +1,68 @@
+=========================
+Release 327 (16 Dec 2019)
+=========================
+
+General Changes
+---------------
+
+* Fix out-of-order data source reads when using ``work_processor_pipelines``. (:issue:`2144`)
+* Fix failure of :func:`word_stem` for certain inputs. (:issue:`2145`)
+* Fix potential incorrect results for queries involving ``FILTER (WHERE ...)`` when the condition
+  is a reference to a table column. (:issue:`2267`)
+* Add rename view. (:issue:`1060`)
+* Add execution time statistics for event listeners. (:issue:`2247`)
+* Add ``error_type`` and ``error_code`` columns to ``system.runtime.queries``. (:issue:`2249`)
+* Rename ``experimental.work-processor-pipelines`` configuration property to ``experimental.late-materialization.enabled``
+  and rename ``work_processor_pipelines`` session property to ``late_materialization``. (:issue:`2275`)
+
+Security Changes
+----------------
+
+* Add support for configuration of multiple system access controls. (:issue:`2178`)
+
+Hive Connector Changes
+----------------------
+
+* Fix incorrect query results when reading ``timestamp`` values from ORC files written by Hive 3.1 or later. (:issue:`2099`)
+* Fix a CDH 5.x metastore compatibility issue resulting in failure when inserting data into, or
+  analyzing a table with ``date`` columns. (:issue:`556`)
+* Reduce number of metastore calls when fetching partitions. (:issue:`1921`)
+* Support reading from insert-only transactional tables. (:issue:`576`)
```

Comment:
Let's add "ACID" before insert-only so it's more clear what this is about

Reply:
I think transactional tables are
* "full acid"  (you can insert, update, delete, select)
* "insert-only" (you can insert or select)

i would be cautious before using "acid insert-only", as this may be easily misunderstood.

Reply:
Well, "ACID" refers to "atomicity/consistency/isolation/durability", so it's orthogonal to insert-only or fully-updatable. I do think it's a bit redundant vs "transactional", since transactional in this context typically refers to having those properties.

Reply:
> "ACID" refers to "atomicity/consistency/isolation/durability"

@martint of course. I was referring to Hive parlance here, as this is the context in which this item will be interpreted.